app/middleware/service-switcher.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const logger = require('../utils/logger')(__filename)
+const validAccountId = require('../utils/valid_account_id.js')
+const { renderErrorView } = require('../utils/response.js')
+
+const notAuthorised = (req, res) => {
+  return renderErrorView(req, res, 'You do not have the rights to access this service.', 403)
+}
+
+module.exports = (req, res, next) => {
+  let newAccountId = req.params.serviceId
```

Comment:
Does the path contain a service id or a gateway account id? I'm confused.
Also this should be a const.